### PR TITLE
support identity link

### PR DESCRIPTION
**Pull request by qs-wang**

### CODE REVIEW

File: user-task.go [hunk 1]
```diff
@@ -74,6 +86,36 @@ func (t *UserTask) Complete(query QueryUserTaskComplete) error {
 	return nil
 }
 
+// GetIdentityLinks retrieve IdentityLinks of the UserTask
+func (t *UserTask) GetIdentityLinks() (*[]IdentityLink, error) {
+	links, err := t.api.GetIdentityLinks(t.Id)
+	if err != nil {
+		return nil, fmt.Errorf("can't complete task: %w", err)
```
Comment (liderman) [inline user-task.go]: looks like copy paste.
I think it's worth doing this:
```go
return nil, fmt.Errorf("can't get identity links: %w", err)
```

File: user-task.go [hunk 2]
```diff
@@ -74,6 +86,36 @@ func (t *UserTask) Complete(query QueryUserTaskComplete) error {
...
+	}
+
+	return links, nil
+}
+
+// AddIdentityLink add IdentityLink to UserTask
+func (t *UserTask) AddIdentityLink(query ReqIdentityLink) error {
+	err := t.api.AddIdentityLink(t.Id, query)
+	if err != nil {
+		return fmt.Errorf("can't complete task: %w", err)
```
Comment (liderman) [inline user-task.go]: I think it's worth doing this:
```go
return nil, fmt.Errorf("can't add identity link: %w", err)
```

File: user-task.go [hunk 3]
```diff
@@ -74,6 +86,36 @@ func (t *UserTask) Complete(query QueryUserTaskComplete) error {
...
+	}
+
+	return nil
+}
+
+// DeleteIdentityLink delete IdentityLink from UserTask
+func (t *UserTask) DeleteIdentityLink(query ReqIdentityLink) error {
+	err := t.api.DeleteIdentityLink(t.Id, query)
+	if err != nil {
+		return fmt.Errorf("can't complete task: %w", err)
```
Comment (liderman) [inline user-task.go]: I think it's worth doing this:
```go
return fmt.Errorf("can't delete identity link: %w", err)
```

**Review by liderman — CHANGES_REQUESTED** [review]

**Comment by qs-wang** [comment]
@liderman thanks for the review. Messages are fixed.

**Comment by liderman** [comment]
@qs-wang thanks for the fixes.
Changes included in release [v3.3.0](https://github.com/citilinkru/camunda-client-go/releases/tag/v3.3.0)

**Pull request merged**